packages/blade/ios/bladeTests/bladeTests.m

```diff
@@ -0,0 +1,65 @@
+#import <UIKit/UIKit.h>
```

Comment:
Lets remove these tests?

Reply:
do you want me to remove the entire directory `bladeTests`?

Reply:
Yeah lets do that